src/pack-externals.ts

```diff
@@ -91,7 +91,7 @@ function getProdModules(
       !packageJson.devDependencies[externalModule.external]
     ) {
       this.options.verbose &&
```

Comment:
Could we include this in the `buildServerlessV3LoggerFromLegacyLogger` and remove these `this.options.verbose &&` statements?

```
export const buildServerlessV3LoggerFromLegacyLogger = (
  legacyLogger: (text: string) => void;
  verbose: boolean;
): ServerlessPlugin.Logging['log'] => ({
  error: legacyLogger,
  warning: legacyLogger,
  notice: legacyLogger,
  info: legacyLogger,
  debug: legacyLogger,
  verbose: verbose && legacyLogger || () => {},
  success: legacyLogger,
});
```
Something like this? What do you think?